src/MemObject.h

```diff
@@ -36,13 +36,17 @@ class MemObject
     static size_t inUseCount();
 
     void dump() const;
-    MemObject();
+    MemObject(char const *, char const *, const HttpRequestMethod &);
+
     ~MemObject();
 
     /// sets store ID, log URI, and request method; TODO: find a better name
+    /// XXX: remove this method and make corresponding URI fields constant
+    /// when another XXX within MemStore::get() is addressed.
     void setUris(char const *aStoreId, char const *aLogUri, const HttpRequestMethod &aMethod);
 
-    /// whether setUris() has been called
+    /// Whether entry StoreID was provided.
+    /// TODO: probably misnamed.
```

Comment:
If the sketch described in another change request is implemented, then the old setUri() documentation will remain valid because setUris() will only be called with a set/non-nil trio.

Reply:
Done.